src/dapp-tests/Makefile

```diff
@@ -1,3 +1,7 @@
-all:; ./run-tests
-
+all:;
+	dapp install ds-test
+	./run-tests
+	dapp uninstall ds-test
+	git reset HEAD^^
```

Comment:
This is not super pretty. It also reinforces the coupling between dapp and git.

I'll see if making a derivation out of `ds-test` works.

Reply:
I wonder if this can be made with the nix function `solidityPaclage`